src/components/Text/_Text.scss

```diff
@@ -0,0 +1,47 @@
+.inputField {
+  @include field-padding-and-margin(1rem 1rem, .5rem);
+  outline: none;
+  &:hover {
+    border: 2px solid gray;
+  }
+  &:focus,
+  &:active {
+    border: 2px solid black;
+  }
+}
+
+.inputField-error {
+  @include field-padding-and-margin(1rem 1rem, .5rem);
+  outline: none;
+  background-color: $error-color;
+  &:hover {
+    border: 2px solid gray;
+  }
+  &:focus,
+  &:active {
+    border: 2px solid red;
+    background-color: white;
+  }
+}
+
+.inputContainer {
```

Comment:
Icon isn't fixed inside the text box, possibly due to using a percent size here.